File: 3813-VowelConsonantScore/3813-VowelConsonantScore.py
# Last updated: 9/12/2026, 7:17:57 AM
class Solution(object):
    def vowelConsonantScore(self, s):
        c=0
        v=0
        for i in s:
            if i in "aeiou":
                v+=1
            elif i.isalpha():
                c+=1
        if c==0:
            return 0
        else :
            return v//c

File: 3813-VowelConsonantScore/3813-VowelConsonantScore.java
// Last updated: 9/12/2026, 7:18:07 AM
class Solution {
    public int vowelConsonantScore(String s) {
        int c=0,v=0;
        for(char a:s.toCharArray()){
            if("aeiou".contains(String.valueOf(a))){
                v++;
            }else if(Character.isLetter(a)){
                c++;
            }
        }
        if(c==0){
            return 0;
        }
        return (int)Math.floor(v/c);
    }
}